designs/sam_destroy_cmd.md

```diff
@@ -0,0 +1,180 @@
+SAM Destroy Command
+====================
+
+This is the design for a command to **destroy** a CloudFormation stack.
+
+What is the problem?
+--------------------
+
+Customers create CloudFormation stacks using `sam deploy`. After deploying the stacks, customers may have to delete
+a stack to cleanup all of their resources and stacks because of stacks that landed in ROLLBACK_FAILED, 
+rapid testing of development tasks, and removing defective code that may have a large security risk.
+
+Currently, Customers would have to open up the CloudFormation console every time they have to delete, forcing them to leave the cli or use the `aws cloudformation` command. 
+This worsens developer experience as it requires them to leave their sam workflow to find what they are looking for. 
+
+ 
+What will be changed?
+---------------------
+
+In this proposal, we will be providing a new command, `sam destroy`, to
+destroy CloudFormation stacks in the cloud. 
+Customers will be able to view failures of the commands and other events directly inside the cli.
+
+
+Success criteria for the change
+-------------------------------
+- Customers will be able to delete a stack in the cloud
```

Comment:
Do we need any constraints on the broadness of the destroy command? eg: --force to delete a stack even if it means there are resources that might be left behind?

Reply:
Do you mean ignore resources that have receive a DELETE_FAILED state?